core/web/terra_transfer_controller.go

```diff
@@ -0,0 +1,122 @@
+package web
+
+import (
+	"fmt"
+	"net/http"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	bank "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/gin-gonic/gin"
+	"github.com/pkg/errors"
+
+	"github.com/smartcontractkit/chainlink-terra/pkg/terra/client"
+
+	"github.com/smartcontractkit/chainlink/core/services/chainlink"
+	terramodels "github.com/smartcontractkit/chainlink/core/store/models/terra"
+	"github.com/smartcontractkit/chainlink/core/web/presenters"
+)
+
+// TerraTransfersController can send LINK tokens to another address
+type TerraTransfersController struct {
+	App chainlink.Application
+}
+
+// Create sends Luna and other native coins from the Chainlink's account to a specified address.
+func (tc *TerraTransfersController) Create(c *gin.Context) {
+	var tr terramodels.SendRequest
+	if err := c.ShouldBindJSON(&tr); err != nil {
+		jsonAPIError(c, http.StatusBadRequest, err)
+		return
+	}
+	if tr.TerraChainID == "" {
+		jsonAPIError(c, http.StatusBadRequest, errors.New("missing terraChainID"))
+		return
+	}
+	chain, err := tc.App.GetChains().Terra.Chain(tr.TerraChainID)
+	switch err {
+	case ErrInvalidChainID, ErrMultipleChains, ErrMissingChainID:
```

Comment:
multiple chains is impossible now with that == "" check right?

Reply:
actually none of these errs are returned from Terra.Chain ?

Reply:
Yeah, lifted from EVM. Do we want to support a 'default' terra chain ID? If not, then this can all be simpler.

Reply:
yeah I'd say not, can just make it explicit